Signal/src/ViewControllers/MediaTileViewController.swift

```diff
@@ -68,7 +68,7 @@ public class MediaTileViewController: UICollectionViewController, MediaGalleryDa
 
         collectionView.backgroundColor = Theme.backgroundColor
 
-        collectionView.register(MediaGalleryCell.self, forCellWithReuseIdentifier: MediaGalleryCell.reuseIdentifier)
+        collectionView.register(PhotoGridViewCell.self, forCellWithReuseIdentifier: PhotoGridViewCell.reuseIdentifier)
```

Comment:
Media cells can represent images, videos and animated images, right?

Reply:
My point is - is photo the best name?

Reply:
I personally don't think it's surprising that the **Photo**Grid includes videos and animated gifs. e.g. see the **Photo**s app.